workbook/04-contributor.asciidoc

```diff
@@ -0,0 +1,178 @@
+== Video:  InnerSource Contributor
+
+=== SEGMENT: Becoming an InnerSource Contributor
+
+TIP:
+More than one answer may be correct in some questions.
+
+==== Question 1.  What is a benefit of becoming a contributor to an InnerSource project?
+
+. I can make sure others have to follow my suggestions to improve the project.
+. The host team will maintain the features I add on behalf of my team.
+. I can shape the solution to better fit my team’s needs.
+. I can help to shape all aspects of the project beyond the code itself (for example, GitHub branches and merges, tests, documentation).
```

Comment:
I'm not sure, GitHub merges are generally something that contributors would do. Though there are examples in the open source world of contributors and host members to share review load, write access in my experience is still reserved for the host.

https://blogs.apache.org/comdev/entry/an-approach-to-community-building ... check section "review then commit"

Reply:
Maybe clarify by "merge to master" or removing the merges?
```suggestion
. I can help to shape all aspects of the project beyond the code itself (for example, GitHub reviewing, bug triage, tests, documentation).
```

Reply:
LGTM